src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/FallThroughCheck.java

```diff
@@ -36,8 +36,9 @@
  * </p>
  * <p>
  * The check honors special comments to suppress the warning.
- * By default the text "fallthru", "fall through", "fallthrough",
- * "falls through" and "fallsthrough" are recognized (case sensitive).
+ * By default the texts "fallthru", "fall through", "fallthrough",
+ * "falls through", "fallsthrough", "fall-through" and other variants
+ * are recognized (case sensitive).
```

Comment:
> and other variants are recognized

We should be specific about what is recognized. Someone may think an underscore should be a recognized variant.

Reply:
I will change regexp to simply sequence all possible , it will more user friendly and easy to maintain, but might be long.

Reply:
regexp stay the same, I updated texts variants